errors/internal.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"errors"
+)
+
+//nolint
+var (
+	ErrInternalInvalidDockerFile = errors.New("invalid Dockerfile")
```

Comment:
Since these are internal errors, these can simply be in `checks` package and not be exported?

Reply:
good point. I initially put them all in `errors/` so they are all in one place, but I think we can move those to `checks/` unless we think they may be shared with other parts of the code, e.g. `pkg/`.